book/go/api/api.md

```diff
@@ -66,7 +66,7 @@ Add a stateless computation that only returns one message to the current pipelin
 
 Similar to `To`, but the computation can return more than one message.
 
-##### `ToStatePartition(stateComputation wallarooapi.StateComputation, stateBuilder wallarooapi.StateBuilder, stateName string, partitionFunction wa.PartitionFunction, partitions []uint64) *pipelineBuilder`
+##### `ToStatePartition(stateComputation wallarooapi.StateComputation, stateBuilder wallarooapi.StateBuilder, stateName string, partitionFunction wa.PartitionFunction, partitions []byte) *pipelineBuilder`
```

Comment:
`ToStatePartition` no longer takes a `partitions` argument, but we've added `ToStatePartitionWithKeys` which still takes the `partitions` argument.

The same thing applies to `ToStatePartitionMulti`. That method no longer takes a `partitions` argument but we've added `ToStatePartitionMultiWithKeys`.